packages/darklang/prettyPrinter/programTypes.dark

```diff
@@ -400,8 +508,9 @@ module Darklang =
 
         | EPipeFnCall(_id, fnName, typeArgs, args) ->
           let fnNamePart =
-            PACKAGE.Darklang.PrettyPrinter.ProgramTypes.FnName.fullForReference
-              fnName
+            match fnName with
+            | Ok name -> PACKAGE.Darklang.PrettyPrinter.ProgramTypes.FnName.fullForReference name
+            | Error e -> String.join e.names "."
```

Comment:
I guess we might as well extract out a helper fn at the top of this file for NameResolution.Error, and reference it where relevant

something like
```fsharp
let nameResolutionError (nr: ...NameResolution): string = String.join e.names "."
```